src/dreyfus_index.erl

```diff
@@ -40,6 +40,8 @@
     waiting_list=[]
 }).
 
+-define(INDEX_SHUTDOWN_DELAY, 60000). % 1 minute
```

Comment:
looks like we don't have any config for dreyfus, if I make this configurable then do we need to add a new config file?
Not sure if it's worth the effort to make this configurable.